pkg/backup/backup.go

```diff
@@ -116,9 +128,9 @@ func NewBackupController(kclient kubernetes.Interface, clusterName, ns string, s
 	}
 
 	return &BackupController{
-		listenAddr:    listenAddr,
+		listenAddr:    config.ListenAddr,
 		backupNow:     make(chan chan backupNowAck),
-		policy:        *sp.Backup,
+		policy:        *bp,
```

Comment:
This could cause nil reference

Reply:
if we ensure bp can't not be nil, then the this will not be a problem.